tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_friendship.c

```diff
@@ -303,8 +305,49 @@ static void test_friend_overflow(void)
 	 */
 	bt_mesh_test_send(friend_lpn_addr, 5, 0, K_NO_WAIT);
 
-	ASSERT_OK(evt_wait(FRIEND_POLLED, K_SECONDS(35)),
-		  "Friend never polled");
+	ASSERT_OK_MSG(evt_wait(FRIEND_POLLED, K_SECONDS(35)),
+		      "Friend never polled");
+
+	/* LPN verifies that no more messages are in Friend Queue. */
+	k_sleep(K_SECONDS(10));
+
+	LOG_INF("Testing overflow with unsegmented message preempting segmented one...");
+
+	/* Give the space in Friend Queue for one unsegmented message. */
```

Comment:
```suggestion
	/* Make room in the Friend Queue for only one unsegmented message. */
```
Does this have the right meaning. The original comment is a little ambigious to me